action.yml

```diff
@@ -0,0 +1,62 @@
+name: action-validator
+description: Downloads, caches, and runs action-validator with verbose output on files tracked by Git.
+inputs:
+  version:
+    description: action-validator version in semver format without the 'v' prefix
+    required: false
+    default: "0.8.0"
```

Comment:
I don't suppose there's a way we can automatically detect the "latest" version of action-validator, and use that by default?  I'm terrible at remembering to update a version number in eleventy billion different places when I make a release.